Formula/qtwebkit.rb

```diff
@@ -0,0 +1,49 @@
+class Qtwebkit < Formula
+  desc "Qt port of WebKit"
+  homepage "https://trac.webkit.org/wiki/QtWebKit"
+  url "https://download.qt.io/official_releases/qt/5.9/5.9.1/submodules/qtwebkit-opensource-src-5.9.1.tar.xz"
+  sha256 "28a560becd800a4229bfac317c2e5407cd3cc95308bc4c3ca90dba2577b052cf"
+
+  keg_only <<-EOS.undent
```

Comment:
should use `:provided_by_osx`

Reply:
Or `should use :provided_by_macos`? Not sure if the plan is to change them at some point or keep both around indefinitely?

Reply:
lol. I didn't even know we had two … :see_no_evil:

Reply:
Aye. Some of the docs over in the `brew` repo explicitly recommends `keg_only :provided_by_macos` as well 🙈. The language itself has been around since https://github.com/Homebrew/brew/commit/3f9cce0a03e967d2e7bcb7cd16bbc898c1a35708.

Reply:
Maybe they rename back to OS X (iPhone X and all) for 10.14 so I can stop being confused.

Reply:
Since @annulen said
>Please note that https://download.qt.io/official_releases/qt/5.9/5.9.1 contains old QtWebKit

So I think that everyone should know that System's WebKit which was built from WebKitGTK+ is more secure.